ix-dev/stable/nextcloud/questions.yaml

```diff
@@ -307,7 +306,7 @@ questions:
                 description: The external port for Nginx.
                 schema:
                   type: int
-                  default: 443
+                  default: 30027 # 443 used by host
```

Comment:
Why is this default better?

Reply:
443 is the **external** port. ie a port in the firewall or some other proxy on the network, not always on the host.